pysorter/filetypes.py

```diff
@@ -140,6 +141,9 @@ def impossible(match, path):
     (r'\.iso$', 'disc_images/iso/'),
     (r'\.xpi$', 'firefox/extensions/'),
     (r'\.jad$', 'javame/jads/'),
+    (r'\.fnt$', 'fonts/'),
+    (r'\.fon$', 'fonts/'),
+    (r'\.otf$', 'fonts/open_type/'),
```

Comment:
Maybe we should just place all the fonts inside the `fonts/` directory (including ttf), what do you think?
I'll merge after this update